src/routes/_components/status/StatusToolbar.html

```diff
@@ -67,6 +67,11 @@
       height: 20px;
     }
   }
+
+  /* Hack, do this properly with state... */
```

Comment:
Just hacked this so people could see how it looks...

Reply:
No worries, I fixed it to use state: 6fcdb149

I can't recall why I made `isStatusInNotification` false if the notification is a mention. I think it was to give a different style to mentions vs other kinds of notifications. In any case I just added `isStatusInNotificationIncludingMentions` and it works. :man_shrugging: